pkg/ipamd/datastore/data_store.go

```diff
@@ -663,51 +628,52 @@ func (ds *DataStore) AssignPodIPv4Address(ipamKey IPAMKey) (ipv4address string,
 	for _, eni := range ds.eniPool {
 		ds.log.Debugf("Checking for eni %v", eni)
 		if !eni.IsPDEnabled {
-			for _, addr := range eni.IPv4Addresses {
-				if !addr.Assigned() && !addr.inCoolingPeriod() {
-					ds.assignPodIPv4AddressUnsafe(ipamKey, eni, addr)
-					if err := ds.writeBackingStoreUnsafe(); err != nil {
-						ds.log.Warnf("Failed to update backing store: %v", err)
-						// Important! Unwind assignment
-						ds.unassignPodIPv4AddressUnsafe(addr)
-						return "", -1, err
+			for _, assignedaddr := range eni.AvailableIPv4Cidrs {
+				ds.log.Debugf("DUMP Assigned addr %v", assignedaddr)
```

Comment:
DUMP --> for debug purposes?

Reply:
Yeah, will clean it up in the later PR. Need it for few more iterations.